api/tests/Api/Users/ListUsersTest.php

```diff
@@ -21,7 +21,7 @@ public function testListUsersIsAllowedForLoggedInUserButFiltered() {
         $response = static::createClientWithCredentials()->request('GET', '/users');
         $this->assertResponseStatusCodeSame(200);
         $this->assertJsonContains([
-            'totalItems' => 1,
+            'totalItems' => 8,
```

Comment:
Ist die Frage, ob wir dennoch uneingeschränktes Auflisten aller User ohne Einschränkung mittels Suchparameter oder ähnlichem erlauben wollen. Wir geben jetzt keine wesentlichen Daten mehr raus, aber ist halt dennoch ein zusätzlicher Attack-Vektor, wenn man schnell alle User auflisten kann. Kann von mir aus aber gerne in eine separate Diskussion bzw. separates Issue.

Reply:
Wir könnten die collection GET Operation abschalten. Evtl wollen wir mal noch nach usern suchen, können es ja dann wieder einschalten. Wir brauchen momentan nur die item GET Operation um referenzen aufzulösen.

Meinungen?

Reply:
Ich glaube fürs MVP brauchen wir dann sowieso eine Suchfunktion, womit die GET Collection Operation wieder nötig wird. Daher würde ich sie jetzt nicht extra nochmals entfernen.
Ich verstehe die Frage von usu eher so, ob es möglich ist, das Abrufen aller User ohne jegliche Filter zu verbieten, aber das Abrufen mit Query Parametern zu erlauben. So könnte die Angriffsfläche für DoS-Attacken verkleinert werden.

Reply:
Für DoS eher weniger, dann würde man das bei allen Entities machen.
Wahrscheinlich eher um nicht alle User ID's zu leaken.

Ich denke fürs MVP brauchen wir keine Suchfunktion. Diese haben wir ausgebaut, damit wir einen einzigen Ablauf haben, wie ein neues Mitglied ins camp kommt (per E-Mail). Für das MVP denke ich reicht es völlg, wenn man die E-Mail der Leiter in das Input feld kopiert, und so die Leiter einlädt.

Daran denken, dass beim abschalten der Endpoint nicht mehr im HalEntryPointNormalizer drin ist. #2347

Reply:
> Wahrscheinlich eher um nicht alle User ID's zu leaken.
Ja genau, um das geht es mir (UserIDs und auch alle DisplayNames).

> Ich denke fürs MVP brauchen wir keine Suchfunktion.
Ok. für mich. Ich würde in diesem Fall dafür voten, die GET Operation für den Moment auszuschalten.

Reply:
Ist disabled